hugo/content/guides/multiple-languages.md

```diff
@@ -0,0 +1,454 @@
+
+```js
+{
+    "projectName": "MultipleLanguages",
+    "languages": [{
+        "id": "multiple-languages-configuration",
+        "grammar": "src/language/multiple-languages-configuration.langium",
+        "fileExtensions": [".me"],
+        "textMate": {
+            "out": "syntaxes/multiple-languages-configuration.tmLanguage.json"
+        },
+        "monarch": {
+            "out": "syntaxes/multiple-languages-configuration.monarch.ts"
+        }
+    }, {
+        "id": "multiple-languages-definition",
+        "grammar": "src/language/multiple-languages-definition.langium",
+        "fileExtensions": [".who"],
+        "textMate": {
+            "out": "syntaxes/multiple-languages-definition.tmLanguage.json"
+        },
+        "monarch": {
+            "out": "syntaxes/multiple-languages-definition.monarch.ts"
+        }
+    }, {
+        "id": "multiple-languages-implementation",
+        "grammar": "src/language/multiple-languages-implementation.langium",
+        "fileExtensions": [".hello"],
+        "textMate": {
+            "out": "syntaxes/multiple-languages-implementation.tmLanguage.json"
+        },
+        "monarch": {
+            "out": "syntaxes/multiple-languages-implementation.monarch.ts"
+        }
+    }],
+    "out": "src/language/generated"
+}
+```
+
+From now on you are able to run the Langium CLI using the NPM scripts (`npm run langium:generate`). It will generate one file for the abstract syntax tree (AST) containing all languages concepts (it is also a good idea to keep the names of these concepts disjoint).
+
+For the next step you need to run the Langium generator once:
+
+```sh
+npm run langium:generate
+```
+
+### `Djinject` language module
+
+The `Djinject` module describes how your language services are built.
+After adding two more languages, some important classes get generated - which need to be registered properly.
+
+1. Open the module file (`/src/language/multiple-languages-module.ts`).
+2. You will notice a wrong import (which is ok, we renamed it in the previous steps and derived new classes by code generation).
+3. Import the new generated modules instead.
+   Replace this line:
+   ```ts
+   import { 
+     MultipleLanguagesGeneratedModule, 
+     MultipleLanguagesGeneratedSharedModule
+   } from './generated/module.js';
+   ```
+   with the following:
+   ```ts
+   import {
+     MultiConfigurationGeneratedModule,
+     MultiDefinitionGeneratedModule,
+     MultiImplementationGeneratedModule,
+     MultipleLanguagesGeneratedSharedModule
+   } from './generated/module.js';
+   ```
+4. In the function `createMultipleLanguagesServices` you will notice an error line now, because we deleted the old class name by the previous step. The code there needs basically be tripled. But before we do this, we need to define the new output type of `createMultipleLanguagesServices`. In the end this should lead to this definition:
```

Comment:
```suggestion
4. In the function `createMultipleLanguagesServices` you will notice an error line now, because we deleted the old class name in the previous step. The code there needs to basically be tripled. But before we do this, we need to define the new output type of `createMultipleLanguagesServices`. In the end this should lead to this definition:
```